src/api/hooks/useRefetchPerson.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from '@tanstack/react-query';
+import { CristinPerson } from '../../types/user.types';
+import { fetchPerson } from '../cristinApi';
+
+interface UseRefetchPersonProps {
+  userCristinId: string;
+  sideEffect: (data: CristinPerson) => void;
+  errorMessage: string;
+}
+
+export const useRefetchPerson = ({ userCristinId, sideEffect, errorMessage }: UseRefetchPersonProps) => {
+  return useQuery({
+    enabled: false,
+    queryKey: ['currentUser', userCristinId],
```

Comment:
`person` brukes som key for henting av person andre plasser, og er dermed det som er cachet
```suggestion
    queryKey: ['person', userCristinId],
```